ping/README.md

```diff
@@ -0,0 +1,98 @@
+# Agent Check: Ping
+
+## Overview
+
+This check uses the system [ping][1] command to test the reachability of a host.
+It also optionally measures the round-trip time for messages sent from the check to the
+destination host.
+
+Ping operates by sending Internet Control Message Protocol (ICMP) echo request packets
+to the target host and waiting for an ICMP echo reply.
+
+This check uses the system ping command, rather than generating the ICMP echo request
+itself, as creating an ICMP packet requires a raw socket, and creating raw sockets
+requires root privileges, which the Agent does not have. The ping command uses the
+`setuid` access flag to run with elevated privileges, avoiding this issue.
+
+## Setup
+
+The Filebeat check is **NOT** included in the [Datadog Agent][2] package.
+
+### Installation
+
+If you are using Agent v6.8+ follow the instructions below to install the Ping check on your host. See our dedicated Agent guide for [installing community integrations][3] to install checks with the [Agent prior to version 6.8][4] or the [Docker Agent][5]:
+
+1. Install the [developer toolkit][6].
+2. Clone the integrations-extras repository:
+
+    ```
+    git clone https://github.com/DataDog/integrations-extras.git.
+    ```
+
+3. Update your `ddev` config with the `integrations-extras/` path:
+
+    ```
+    ddev config set extras ./integrations-extras
+    ```
+
+4. To build the `ping` package, run:
+
+    ```
+    ddev -e release build ping
+    ```
+
+5. [Download and launch the Datadog Agent][7].
+6. Run the following command to install the integrations wheel with the Agent:
+
+    ```
+    datadog-agent integration install -w <PATH_OF_PING_ARTIFACT_>/<PING_ARTIFACT_NAME>.whl
```

Comment:
For systems with sudo, you need to be the dd-agent user to install a wheel into the embedded python env
```suggestion
    sudo -u dd-agent datadog-agent integration install -w <PATH_OF_PING_ARTIFACT_>/<PING_ARTIFACT_NAME>.whl
```

Reply:
👍 We need to review some other extras integrations then, the filebeat integration was edited a while back using that command without sudo, see: https://github.com/DataDog/integrations-extras/commit/c48a33634b525a96395aba3453284416b85ba8a5#diff-3e34b150aecc763c950d761f89ea18b9R41, cc @l0k0ms .